READMEs/datatokens-flow.md

```diff
@@ -141,4 +141,7 @@ print(f"datatoken name: {erc20_token.token_name()}")
 print(f"datatoken symbol: {erc20_token.symbol()}")
 ```
 
+As an alternative for publishing a NFT and a datatoken at once, you can use `create_nft_with_erc20`.
```

Comment:
I think you can move this to the very end. Give the user their "congrats" first 😅

Reply:
Also, make sure you mention that the arguments for NFT creation are a bit different for the "at once" functions, since the additional erc 20 deployer will automatically be set as the factory router (use the ...NoDeployer named tuple)

Reply:
This applies to all "at once" functions btw